homeassistant/components/flume/sensor.py

```diff
@@ -84,15 +84,10 @@ def state(self):
         """Return the state of the sensor."""
         return self._state
 
-    @property
-    def unit_of_measurement(self):
-        """Return the unit the value is expressed in."""
-        return self._unit_of_measurement
```

Comment:
I was thinking of `return "GALLONS"` instead of a variable (and I think it should be "gal" instead).

Reply:
Restored. I really appreciate all the feedback. Thank you!